setup-environment-internal

```diff
@@ -262,22 +262,40 @@ ln -sf "${MANIFESTS}" "${OEROOT}"/layers/
 
 DISTRO_DIRNAME=$(echo "${DISTRO}" | sed 's#[.-]#_#g')
 
+LMP_TAG="$(git --git-dir ${MANIFESTS}/.git describe HEAD --tags --abbrev=0)"
+LMP_TAG="$(echo $LMP_TAG | sed 's/\.[0-9]*$//')"
+
+if [[ ! $LMP_TAG =~ ^[[:digit:]] ]]; then
+    LMP_TAG_ARR=(${LMP_TAG//-/ })
+    LMP_PARTNER_NAME="${LMP_TAG_ARR[0]}"
+    LMP_VERSION_CACHE_TMP="${LMP_TAG_ARR[1]}"
+else
+    LMP_VERSION_CACHE_TMP="${LMP_TAG}"
+fi
+
 if [ -z "$LMP_VERSION_CACHE" ]; then
-    LMP_VERSION_CACHE="$(git --git-dir ${MANIFESTS}/.git describe HEAD --tags --abbrev=0)"
-    LMP_VERSION_CACHE="$(echo $LMP_VERSION_CACHE | sed 's/\.[0-9]*$//')"
+    LMP_VERSION_CACHE="${LMP_VERSION_CACHE_TMP}"
+
     if [ -v LMP_VERSION_CACHE_DEV ]; then
         # to use the development version of the cache the user need to define the LMP_VERSION_CACHE_DEV env
         LMP_VERSION_CACHE=$(( $LMP_VERSION_CACHE + 1 ))
     fi
 fi
 
+# Use public state cache mirror if no other is defined
```

Comment:
The above comment only make sense for the bellow bitbake assignment `??=` so we can remove it

Reply:
removed the comment, thanks